src/server.ts

```diff
@@ -1,47 +1,98 @@
 import * as fsPath from 'path';
+import * as http from 'http';
 import * as nunjucks from 'nunjucks';
 
 import {Pod} from './pod';
 import {StaticRoute} from './router';
+import {Watcher} from './watcher';
 
 import express = require('express');
 
 export class Server {
-  constructor() {}
-}
+  private pod: Pod;
+  private app?: express.Application;
```

Comment:
is this being read anywhere? i see it being set, but never actually re-read. if not, move it to be a local variable only.

Reply:
Removed